templates/swifty-gtk4/LICENSE.in

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I've tried to avoid including licenses in the templates because I want that to be a matter for the user to decide.